earth_enterprise/src/fusion/gepublish/PublishHelper.cpp

```diff
@@ -156,8 +156,7 @@ bool PublishHelper::ProcessGetRequest(const std::string& url,
   else
     *args += host_name;
 
-  notify(NFY_DEBUG, "Request %s args: %s %s", url.c_str(), args->c_str(),
-         (username_.empty() ? "nouser" : "user"));
+  notify(NFY_DEBUG, "ProcessGetRequest %s Starting", url.c_str());
```

Comment:
It might be a good idea to reverse this change since the older version seems to give more information.

Reply:
Yes the older version did give more but the problem was the caller also traced out the args at NFY_DEBUG level.  Normally double tracing things at debug level isn't a terrible thing but that parameter is huge and just made reading the debug logs difficult having to wade through almost back to back dumps of that argument.  Maybe put it back here and remove it from the caller?

Reply:
I do see I removed the user trace.  Should put that back.

Reply:
I think I'm going to change the caller and put this one back the way it was.

Reply:
I take that back.  This is the caller.  The callee below `request.Start(*args)` also traces the arguments out.  Probably should leave the lower level one in and take the arg trace out of this one as it adds no value.  Actually makes things more difficult.